zipline/data/us_equity_pricing.py

```diff
@@ -778,24 +795,17 @@ def last_available_dt(self):
 
     @property
     def trading_calendar(self):
-        return None
+        return self._trading_calendar
```

Comment:
I made `PanelBarReader` a subclass of `DailyBarReader`, which has`trading_calendar` as an abstract property: https://github.com/quantopian/zipline/blob/master/zipline/data/us_equity_pricing.py#L405